terraform/variables.tf

```diff
@@ -0,0 +1,45 @@
+variable "app_name" {
+  description = "Name to give the deployed application"
+  type        = string
+  default     = "kiali"
+}
+
+variable "channel" {
+  description = "Channel that the charm is deployed from"
+  type        = string
+}
+
+variable "config" {
+  description = "Map of the charm configuration options"
+  type        = map(string)
+  default     = {}
+}
+
+variable "constraints" {
+  description = "String listing constraints for this application"
+  type        = string
+  default     = ""
```

Comment:
See [this thread](https://github.com/canonical/istio-ingress-k8s-operator/pull/95/files#r2237464325)

Reply:
Fixed.